oaknational/Oak-Web-Application
# Feat/lesson overview lesson details component LESQ-110 LESQ-111 LESQ-44

## Lesson Overview Iteration

New Components and test files:
Lesson Details - Created Grid for iteration also
KeyLearningPoints
KeyWords
CommonMisconceptions

Removed Components:
Lesson Overview

Reorders lessonSlug components and adds blue line also

## Issue(s)

Fixes LESQ-110 LESQ-111 LESQ-44

## How to test

1. Go to https://deploy-preview-1759--oak-web-application.netlify.thenational.academy
2. Click on  
/beta/teachers-2023/programmes/science-primary-ks1/units/growing-plants/lessons/lesson-1-in-growing-plants
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2023-07-25 at 14 13 16](https://github.com/oaknational/Oak-Web-Application/assets/91190841/e5abf57f-84ff-4bfe-b2bd-c935eaba5acb)

How it should now look:
![Screenshot 2023-07-25 at 14 11 13](https://github.com/oaknational/Oak-Web-Application/assets/91190841/36721676-c55c-4e0d-a72d-dca2b44c6a56)

## Checklist
